psychopy/tools/filetools.py

```diff
@@ -248,15 +252,18 @@ def path_to_string(filepath):
     This WILL NOT work with on Python 3.4, 3.5 since the __fspath__ dunder
     method did not exist in those verisions, however psychopy does not support
     these versions of python anyways.
-     :Parameters:
-     filepath : string or pathlib Path object
+
+    :Parameters:
+
+    filepath : string or pathlib Path object
```

Comment:
Could you please reference to `pathlib.Path` via `:class:pathlib.Path` please, so we get correct markup via Sphinx?

Reply:
Oh, and `string` should be `str`, ideally :)

Reply:
To cut it short, the line should probably be:
```python
filepath : str or pathlib.Path
```

Reply:
Hmm, I'm not 100% certain either. Should we give it a go and just add in ```:class:`pathlib.Path` ``` so the parameters appear as follows:

    :Parameters:

     filepath : string or :class:`pathlib.Path`

Or do you think we should wait for Jon to weigh in to make sure its done correctly?

Reply:
We could also simply build the docs ourselves and check ;) But I'm currently a little busy. I think I would try without the `:class:` markup for now… :)

Reply:
The change I made to confy.py in a4de786574ea3155681b6cb4db48107bbe6b619d makes it possible to use :class:`pathlib.Path` (needed the addition of intersphinx to the config file and then an instruction for how to find the docs of pathlib